reference/hoon-expressions/rune/bar.md

```diff
@@ -8,12 +8,78 @@ Core expressions produce cores. A core is a cell of `[battery payload]`.
 The `battery` is code, a battery of Nock formulas.  The `payload` is the data
 needed to run those formulas correctly.
 
-Four core runes (`|=`, `|.`, `|-`, and `|*`) produce a
+Five core runes (`|=`, `|.`, `|-`, `|*`, and `|$`) produce a
 core with a single arm, named `$`. We can recompute this arm
-with changes, which useful for recursion among other things:
+with changes, which is useful for recursion among other things.
 
 ## Runes
 
+### |$ "barbuc"
+
+Declares a mold builder wet gate with one or more molds as its
+sample.
```

Comment:
A "mold builder" is simply a wet gate from one-or-more molds to a mold. A mold is a (maybe) partial, (definitely) idempotent, (usually) identity  function from noun to some more specific type.

Reply:
How's this? https://github.com/urbit/docs/pull/877/commits/1a0e1a4616736a323a80eddc823a049b9cfe0cf0